src/gsi/hybrid_ensemble_isotropic.F90

```diff
@@ -1393,6 +1393,10 @@ subroutine load_ensemble
                 endif
     
 !     regional_ensemble_option = 5: ensembles are fv3 regional.
+                if (l_both_fv3sar_gfs_ens) then ! first read in gfs ensembles for regional
```

Comment:
for line 1394 to 1399, why do we need to change to it?   though the essential change is  of the comment. 
I believe the "cltthink .." question is not needed now.